src/index.ts

```diff
@@ -106,9 +84,9 @@ program
       options: ${b(JSON.stringify(options))}
       program: ${b(data)}
     `)
-    const { dryRun } = options
+    const { dryRun, verbose } = options
     try {
-      const res: Process.Response = await loadProcesses({ data, dryRun, options: mapOptions(options) })
+      const res: Process.Response = await loadProcesses({ data, dryRun, options: mapOptions(options), verbose })
```

Comment:
Maybe we could include verbose in mapOptions? less places to update, would need to go through to confirm tho as it looks like polkadot related only. https://github.com/digicatapult/dscp-process-management/blob/83f0e6d45775ee86202367c53a5194390b094d9c/src/index.ts#L28C1-L32C3

```
const mapOptions = (options: Process.CLIOptions): Polkadot.Options => ({
  API_HOST: options.host,
  API_PORT: parseInt(options.port),
  USER_URI: options.user,
})
```

Reply:
I'll leave it for now, and focus on getting the tests done so we can merge.